CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ endif()
 option(WITH_GZFILEOP "Compile with support for gzFile related functions" ON)
 option(ZLIB_COMPAT "Compile with zlib compatible API" OFF)
 option(ZLIB_ENABLE_TESTS "Build test binaries" ON)
+option(ZLIB_COMPAT_TESTS "Test only zlib compatible API" OFF)
```

Comment:
The option name is a bit confusing. What do you think of something like `ZLIBNG_ENABLE_TESTS` and defaulted to `ON`?

Reply:
Ok, I'll rename it.